app/Transformers/ArtworkTransformer.php

```diff
@@ -39,6 +39,7 @@ protected function getFields(Datum $datum)
             'artwork_dates' => $this->mapToArray($datum->artwork_dates, 'getArtworkDate'),
             'artwork_catalogues' => $this->mapToArray($datum->artwork_catalogues, 'getArtworkCatalogue'),
 
+            'dimensions_detail' => $datum->dimensions_detail,
```

Comment:
Generally, we prefer to return `null` if a numeric value is `0`. We have the `nullZero()` helper method that applies this behavior to individual fields, but I think we'd want to apply that to the values within the `dimensions_detail` array in this case.

I also wonder if we don't need both `cm` and `in` in the output. That's fairly simple math that implementers can do on their own. Maybe only include the `cm`s to make the output tighter.